plugins/tekton/src/types.ts

```diff
@@ -0,0 +1,353 @@
+import {
+  TektonParam,
+  TektonResource,
+  TektonResourceGroup,
+  TektonResultsRun,
+  TektonTaskSpec,
+  TektonWorkspace,
+} from './components/core-tekton';
+
+export enum ComputedStatus {
+  Cancelling = 'Cancelling',
+  Succeeded = 'Succeeded',
+  Failed = 'Failed',
+  Running = 'Running',
+  'In Progress' = 'In Progress',
+  FailedToStart = 'FailedToStart',
+  PipelineNotStarted = 'PipelineNotStarted',
+  Skipped = 'Skipped',
+  Cancelled = 'Cancelled',
+  Pending = 'Pending',
+  Idle = 'Idle',
+  Other = '-',
+}
+
+export enum SucceedConditionReason {
+  PipelineRunCancelled = 'StoppedRunFinally',
+  PipelineRunStopped = 'CancelledRunFinally',
+  TaskRunCancelled = 'TaskRunCancelled',
+  Cancelled = 'Cancelled',
+  PipelineRunStopping = 'PipelineRunStopping',
+  PipelineRunPending = 'PipelineRunPending',
+  TaskRunStopping = 'TaskRunStopping',
+  CreateContainerConfigError = 'CreateContainerConfigError',
+  ExceededNodeResources = 'ExceededNodeResources',
+  ExceededResourceQuota = 'ExceededResourceQuota',
+  ConditionCheckFailed = 'ConditionCheckFailed',
+  CouldntCreateWorkspacePVC = 'CouldntCreateWorkspacePVC',
+}
```

Comment:
This looks duplicate of https://github.com/janus-idp/backstage-plugins/pull/242/files#diff-757032c6d81b5bbcf881dd8c91f105bfc798cf32640e6897ff8b68c0947386f4R18-R30